src/System.Text.Json/tests/WritableJsonApiTests.cs

```diff
@@ -624,6 +966,324 @@ public static void TestAquiringAllProperties()
             };
 
             IEnumerable<JsonNode> fullTimeEmployees = employees.GetAllProperties("FTE");
+
+            Assert.Equal(3, fullTimeEmployees.Count());
+            Assert.True(fullTimeEmployees.Contains((JsonString)"John Smith"));
+            Assert.True(fullTimeEmployees.Contains((JsonString)"Ann Predictable"));
+            Assert.True(fullTimeEmployees.Contains((JsonString)"Byron Shadow"));
+        }
+
+        /// <summary>
+        /// Transforming JsoneNode to JsonElement
```

Comment:
NIT: Could we move these Transformation tests into a separate file, this one is already too big I believe. It would also be nice if you move all of the non-fact methods into a .TestData.cs file.

Reply:
Ok, sure, I can group test methods into more categories in separate files.